src/f32-rwsum/f32-rwdsum-1p1x-scalar-c1.c

```diff
@@ -0,0 +1,79 @@
+// Copyright 2024 Google LLC
+//
+// This source code is licensed under the BSD-style license found in the
+// LICENSE file in the root directory of this source tree.
+
+#include <assert.h>
+
+#include "xnnpack/common.h"
+#include "xnnpack/reduce.h"
+
+#define CEILING_POS(X) ((X-(int)(X)) > 0 ? (int)(X+1) : (int)(X))
```

Comment:
These macros should be helper functions in math.h. I think we already have some of the functions you need, e.g. `divide_round_up` (we shouldn't divide first and then try to compute the ceiling of the result, that approach assumes approximating the result with float or fixed point arithmetic).